node/config/types.go

```diff
@@ -52,6 +52,32 @@ type StorageMiner struct {
 	DAGStore   DAGStoreConfig
 }
 
+type DealmakingConfiger interface {
+	GetDealmakingConfig() DealmakingConfig
+	SetDealmakingConfig(DealmakingConfig)
+}
+
+func (c *StorageMiner) GetDealmakingConfig() DealmakingConfig {
+	return c.Dealmaking
+}
+
+func (c *StorageMiner) SetDealmakingConfig(other DealmakingConfig) {
+	c.Dealmaking = other
+}
+
+type SealingConfiger interface {
+	GetSealingConfig() SealingConfig
+	SetSealingConfig(SealingConfig)
+}
+
+func (c *StorageMiner) GetSealingConfig() SealingConfig {
+	return c.Sealing
+}
+
+func (c *StorageMiner) SetSealingConfig(other SealingConfig) {
+	c.Sealing = other
+}
+
```

Comment:
This file should only contain config structs with docstrings, putting other stuff here will mess with config docsgen

Reply:
(see the comment on top of the file)